scripts/dump_meta_data.sql

```diff
@@ -9,6 +9,9 @@
 \echo 'Date, git commit, and extension version can change without it being an error.'
 \echo 'Adding this tag allows us to run regression tests on this script file.'
 \echo `date`
+\echo 'Postgres version'
+select version();
```

Comment:
I think also the extension version is missing.

Reply:
Do you want something more than `\dx` ?

Reply:
Missed that it was in there. Was looking for something like SELECT * FROM pg_extension...